### PR TITLE
Add not found page and connect it to router

### PR DESCRIPTION
**Pull request by EugeniusGR**
when / : 
image

when /sign-in: 
image

when /random-text:
image

### CODE REVIEW

**Comment by EugeniusGR** [comment]
[Create not-found page](https://trello.com/c/NtKIDy7s/21-create-not-found-page)

File: frontend/src/components/common/not-found/notFound.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import logo from '../../../assets/img/logo-cut.svg';
```
Comment (ghost) [inline frontend/src/components/common/not-found/notFound.tsx]:
```suggestion
import logo from 'assets/img/logo-cut.svg';
```

File: frontend/src/components/common/not-found/notFound.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <span className={styles.headerText}>404</span>
+        <span className={styles.descriptionText}>Page not found</span>
+      </div>
+      <footer className={styles.copyRight}>
+        <img
+          src={logo}
+          width="103"
+          height="35"
+          loading="lazy"
+          alt=""
```
Comment (ghost) [inline frontend/src/components/common/not-found/notFound.tsx]:
```suggestion
          alt="Jabber Logo"
```

File: frontend/src/components/common/not-found/notFound.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import styles from './styles.module.scss';
+
+const NotFound: React.FC = () => {
+  return (
+    <div>
+      <div className={styles.textWrapper}>
+        <span className={styles.headerText}>404</span>
+        <span className={styles.descriptionText}>Page not found</span>
+      </div>
+      <footer className={styles.copyRight}>
```
Comment (ghost) [inline frontend/src/components/common/not-found/notFound.tsx]:
```suggestion
      <p className={styles.copyRight}>
```

File: frontend/src/components/common/not-found/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
+.textWrapper {
+  height: 80vh;
```
Comment (ghost) [inline frontend/src/components/common/not-found/styles.module.scss]:
```suggestion
```

File: frontend/src/components/common/not-found/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.descriptionText {
+  display: block;
+  font-size: 50px;
+  font-weight: 600;
+}
+
+.copyRight {
+  height: 7vh;
```
Comment (ghost) [inline frontend/src/components/common/not-found/styles.module.scss]:
```suggestion
```

File: frontend/src/components/common/not-found/notFound.tsx (hunk 1 above)
Comment (ghost) [inline frontend/src/components/common/not-found/notFound.tsx]: Move this component to `common/not-found/not-found.tsx`

**Review by ghost — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/not-found/logo-cut.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<svg width="311" height="190" viewBox="0 0 311 190" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ghost) [inline frontend/src/components/common/not-found/logo-cut.svg]: move to assests

File: frontend/src/components/common/not-found/notFound.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import logo from './logo-cut.svg';
```
Comment (ghost) [inline frontend/src/components/common/not-found/notFound.tsx]: notFound.tsx =>not-found.tsx

**Review by ghost — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/not-found/styles.module.scss [hunk 8]
```diff
@@ -0,0 +1,34 @@
+.textWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-top: 250px;
+}
+
+.headerText {
```
Comment (yevhenprybytok) [inline frontend/src/components/common/not-found/styles.module.scss:9]: Next time use nested classes

**Review by yevhenprybytok — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**